ui/app/AppLayouts/Chat/stores/RootStore.qml

```diff
@@ -54,7 +54,17 @@ QtObject {
     readonly property PermissionsStore permissionsStore: PermissionsStore {
         activeSectionId: mainModuleInst.activeSection.id
         activeChannelId: root.currentChatContentModule().chatDetails.id
-        chatCommunitySectionModuleInst: chatCommunitySectionModule
+        permissionsModel: root.chatCommunitySectionModule.permissionsModel
+        allTokenRequirementsMet: root.chatCommunitySectionModule.allTokenRequirementsMet
+
+        onCreateOrEditCommunityTokenPermission: {
```

Comment:
please the qt6-compatible handlers (function-based). Otherwise we will have flood of warnings.